src/nectar/cli.py

```
@@ -593,23 +588,26 @@ def updatenodes(show, test, only_https, only_wss):
+                if node["url"] in nodes:
+                    score = float("{0:.1f}".format(node["score"]))
+                    t.add_row([node["url"], node["version"], score])
+            print(t)
+        if not test:
+            hv.set_default_nodes(nodes)
+            hv.rpc.nodes.set_node_urls(nodes)
+            hv.rpc.current_rpc = 0
+            hv.rpc.rpcconnect()
+    except Exception as e:
+        print(str(e))
+        raise e
```

Ruff (0.12.2)

Use `raise` without specifying exception name

Remove exception name

(TRY201)